packages/holocron/__tests__/holocronModule.spec.jsx

```diff
@@ -340,14 +365,28 @@ describe('holocronModule', () => {
     const reducer = () => fromJS({ x: 3 });
     const mergeProps = (stateProps, dispatchProps, ownProps) => {
       const xy = stateProps.moduleState.x * ownProps.y;
-      return Object.assign({}, stateProps, dispatchProps, ownProps, { xy });
+      return {
+        ...stateProps, ...dispatchProps, ...ownProps, xy,
+      };
     };
     const Module = holocronModule({
       name: 'mock-module',
       reducer,
       mergeProps,
-    })(({ moduleState: { x }, y, xy }) => <div>{x} * {y} = {xy}</div>);
-    const mockStore = createStore(state => state, fromJS({ modules: { 'mock-module': { x: 3 } } }));
+    })(({ moduleState: { x }, y, xy }) => (
+      <div>
+        {x}
+        {' '}
+*
+        {' '}
+        {y}
+        {' '}
+=
+        {' '}
```

Comment:
we really need to change this rule. @nellyk can you open a PR to the eslint config when you have a chance?

Reply:
is it related to [indentation](https://github.com/americanexpress/holocron/pull/3/files#r371485456)? (is my comment a duplicate? 😅)

Reply:
ooh yeah agreed

Reply:
https://github.com/americanexpress/eslint-config-amex/pull/8

Reply:
once new version of eslint-config-amex is released i shall revert this